game.py

```diff
@@ -20,7 +20,38 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    user_wins = is_word_guessed(snowman_word, correct_letter_guess_statuses)
+
+    while (
+        (len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES)
+        and (not user_wins)
+    ):
+        user_input = get_letter_from_user(correct_letter_guess_statuses, 
+                                        wrong_guesses_list)
+        
+        if user_input in correct_letter_guess_statuses:
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses[user_input] = True
+
+        else:
+            print(f"The letter {user_input} is not in the word")
+            wrong_guesses_list.append(user_input)
+
+        print_snowman_graphic(len(wrong_guesses_list))
+        print(generate_word_progress_string(snowman_word, 
+            correct_letter_guess_statuses))
+        
+        print(f"Wrong guesses: {wrong_guesses_list}")
```

Comment:
I think that it would okay to group these lines together visually since they are all performing logic that is printing some information to the user. 
```suggestion
        print_snowman_graphic(len(wrong_guesses_list))
        print(generate_word_progress_string(
            snowman_word, 
            correct_letter_guess_statuses))
        print(f"Wrong guesses: {wrong_guesses_list}")
```